src/background/utils/db.js

```diff
@@ -433,7 +447,7 @@ export async function parseScript(src) {
   if (src.position) script.props.position = +src.position;
   buildPathMap(script, src.url);
   await saveScript(script, src.code);
-  fetchScriptResources(script, src);
+  fetchScriptResources(script);
```

Comment:
Does this mean that cache will not be used for `@require`d resources?

Reply:
Yep, the idea is to rely on browser caching. Do you think there are cases where it may be detrimental?

Reply:
This is ideal if the resource has a long time cache. Other than that, this may cause problems for us with bad connections. Even a 304 request could fail if connection cannot be made.

Currently the confirm page allows users to retry loading dependencies by reloading the page. Once the dependencies are loaded, users will expect the installation to be successful. But if the background has to send another request due to different cache controls, the installation might fail.

I have suffered from such cases and that is why I added this cache.

Reply:
Okay, I'll add error handling and fallback on VM's cache.

Reply:
Wait, there should be no problem with the current code. It stores the dependencies in storage only on success so errors or delays on subsequent retries (if tracking is enabled) don't influence anything.

Reply:
Ah, I understand now. There's a duplication of the process in the confimation page and fetchScriptResources.